File: Questions/Bubble sort.cpp
#include<bits/stdc++.h>
#include<climits>
#include<iostream>
using namespace std;


int main(){
	int n;
	cout<<"enter the value"<<endl;
	cin>>n;

	int arr[n];

	for(int i=0; i<n; i++){
		cin>>arr[i];
	}
	int count = 1;
	while(count<n){
		for(int i=0; i<n-count; i++){
			if(arr[i]>arr[i+1]){
				int temp = arr[i];
				arr[i] = arr[i+1];
				arr[i+1] = temp;
			}
		} count++;
	}

	for(int i=0; i<n; i++){
		cout<<arr[i]<<" ";
	} cout<<endl;

}

File: Questions/Java/Bubblesort.java
import java.util.Scanner;

public class test {

    public static void array(int arr[]){
        for(int i=0; i<arr.length; i++){
            System.out.println(arr[i]+ " ");
        }
        System.out.println();
    }

    public static void main(String args[]){
    int arr[] = {4,3,6,5};

    for(int i=0; i<arr.length-1; i++){
        for(int j=0; j<arr.length-i-1; j++){

            if(arr[j]>arr[j+1]){
                int temp = arr[j];
                arr[j] = arr[j+1];
                arr[j+1] = temp;
            }
        } 
    } array(arr); 
    }
    }
